Sprint-1/destructuring/exercise-2/exercise.js

```diff
@@ -70,3 +70,16 @@ let hogwarts = [
     occupation: "Teacher",
   },
 ];
+
+
+const studentHouse = hogwarts.filter(studentHouse => studentHouse.house === "Gryffindor");
+studentHouse.forEach(studentHouse => {
+  console.log(`${studentHouse.firstName} ${studentHouse.lastName} is a student of Gryffindor.`);
```

Comment:
The output has Dumbledore as a _student_ of Gryffindor. Could you modify your code to console out a statement that encompasses all members of Gryffindor?